pages/glossary/embedded-image.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Embedded Image
+key: embedded-image
+unambiguous: true
+objective: true
+input_aspects:
+  - DOM tree
+---
+
+An element presents an _embedded image_ when any of the following is true:
+
+- the element is an `img` element with a non-empty [source set][]; or
+- the element is an `input` element with a `type` [attribute value][] of `image` and its `src` [attribute value][] is not empty; or
+- the element is a `canvas` element with an [image][canvasImage] [drawn][drawImage]; or
+- the element is an `object` element with a `data` [attribute value][] referencing a resource with an [image MIME type](https://mimesniff.spec.whatwg.org/#image-mime-type); or
+- the element is an `svg` element having one or more `image` [descendants][] with a non-empty `href` [attribute value][]; or
```

Comment:
What is the image source of an svg with multiple image elements? I think this need to target the individual image elements, not the svg for this rule to work.

Reply:
The expectation of the rule does target the individual rendered image sources, so it's in accordance to what you're asking.

Reply:
It is not clear to me how you intend for this to work. Perhaps the phrase rendered image resource is ambiguous. I'm not sure, but I don't see how this rule would work in the case of multiple images rendered by the same element. Be it repeating backgrounds, svg with multiple image elements, or canvas with multiple drawn parts.